sections/featured-collection.liquid

```diff
@@ -30,28 +30,32 @@
   endif
 
   assign columns_mobile_int = section.settings.columns_mobile | plus: 0
-  assign show_mobile_slider = false
-  if section.settings.swipe_on_mobile and products_to_display > columns_mobile_int
-    assign show_mobile_slider = true
+  assign show_slider_on_mobile = false
+  if section.settings.enable_mobile_slider and products_to_display > columns_mobile_int
+    assign show_slider_on_mobile = true
+  endif
+
+  assign show_slider_on_desktop = false
+  if section.settings.enable_desktop_slider and products_to_display > section.settings.columns_desktop
+    assign show_slider_on_desktop = true
```

Comment:
Same as above, I think I'd rather have this condition repeated across the file instead of it being part of an `assign`.

This may sound like it improves readability, but personally, I find the idea of always having to go back to the top to check how this variable was formed actually harder to read.